modules/manage/pages/kubernetes/troubleshooting/k-troubleshoot.adoc

```diff
@@ -406,6 +406,82 @@ kubectl logs -f <pod-name> --namespace <namespace>
 You can use the output to debug your deployment.
 //end::deployment-pod-pending[]
 
+//tag::deployment-failed-scheduling[]
+=== FailedScheduling
+
+If you see this error, your cluster does not have enough nodes to satisfy the anti-affinity rules:
+
+[.no-copy]
+----
+Warning  FailedScheduling  18m  default-scheduler  0/1 nodes are available: 1 node(s) didn't match pod anti-affinity rules. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod..
```

Comment:
Does the warning really include two periods at the end? Regardless of how it appears in the actual warning, let's go with a single period at the end. :-)

Reply:
@Feediver1 see https://github.com/redpanda-data/docs/pull/541#discussion_r1631386836